### DISCUSSION TITLE
Problem "CraftingMissions"

### DISCUSSION THREAD

**Discussion by HaazEE77** [BG-Software-LLC/SuperiorSkyblock2#1115] [Approved Feature Requests]
I have a problem with CraftingMissions, I created a category and I can not access it in game, I attach the log
![Sans titre](https://user-images.githubusercontent.com/105245042/167491201-d5c24202-105a-4911-8be7-a355185dbc59.png)
s

**Comment by Lemvs** [comment]
Issues should be reported in Issues section.

@OmerBenGera you can mark this answered and close it.
